tests/integration/testsuites/gateway/testsuite.go

```diff
@@ -15,6 +12,7 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
+	"log"
```

Comment:
I think it's better to handle in follow up as from what I see this would require bigger changes to be made.
It was already there, I just re-ordered imports